### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by binnichtaktiv** [Sangwan5688/Sangwan5688#1]
Hi bro I wanted to ask what happened to Blackhole. It's deleted now.. it was such a good app😭😭😭

**Comment by bazilbycom** [comment]
Yes. We need to know what exactly happened. We will try to help in anyway.

**Comment by bazilbycom** [comment]
Yes. We need to know what exactly happened. We will try to help in anyway.

**Comment by SoCuul** [comment]
> Hi bro I wanted to ask what happened to Blackhole. It's deleted now.. it was such a good app😭😭😭

Well he want through the process of actually asking storefronts to take it down. And that was a month ago, so something definitely happened.

https://gitlab.com/fdroid/fdroiddata/-/issues/3219#note_1824770137
